<?xml version="1.0"?>
<launch>
	<!--arduino node -->
	<node pkg="rosserial_python" type="serial_node.py" name="usb0">
		<param name="port" value="/dev/ttyUSB0"/>
	</node>
    
    <node pkg="rosserial_python" type="serial_node.py" name="usb1">
		<param name="port" value="/dev/ttyUSB1"/>
	</node>
    
    <node pkg="rosserial_python" type="serial_node.py" name="usb2">
		<param name="port" value="/dev/ttyUSB2"/>
	</node>
    
    <node pkg="rosserial_python" type="serial_node.py" name="usb3">
		<param name="port" value="/dev/ttyUSB3"/>
	</node>

	<!--joystick node -->
    <node pkg="joy" type="joy_node" name="joy"/>
    
    <!--computer check node-->
    <node pkg="underwater_robot" type="check_usb" name="check_usb"/>

</launch>
